teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
```

Comment:
- [ ] Kindly include `age`, and `name` in the parameter of the `initialize` method. This would make it possible for you to add custom `names` and `ages` when creating an instance of the `Teacher` class. 
- [ ] In the super method being called in the initialize method, kindly add `name` and `age` to the parameter. `super(name, age)`